site/package.json

```diff
@@ -4,13 +4,14 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "build": "yarn build-blocks && yarn generate-sitemap && next build",
+    "build": "yarn build-blocks && yarn generate-blocks-data && yarn generate-sitemap && next build",
     "build-block": "./scripts/build-blocks.sh",
     "build-blocks": "find ../hub -type f | xargs ./scripts/build-blocks.sh",
     "clean": "rimraf ./.next/",
-    "dev": "yarn generate-sitemap && next dev",
+    "dev": "yarn generate-sitemap && yarn generate-blocks-data && next dev",
     "dev:db": "docker-compose -f docker-compose.dev.yml --env-file .env.local up",
     "dev:seed-db": "node --max-old-space-size=2048 node_modules/.bin/ts-node --files --project tsconfig.json scripts/seed-db",
+    "generate-blocks-data": "node --max-old-space-size=2048 node_modules/.bin/ts-node --files --project tsconfig.json scripts/generate-blocks-data",
     "generate-sitemap": "node --max-old-space-size=2048 node_modules/.bin/ts-node --files --project tsconfig.json scripts/generate-sitemap",
     "postinstall": "yarn generate-sitemap",
```

Comment:
You need to add `yarn generate-blocks-data` here so that the file is available reliably (e.g. in CI)